chat.js

```diff
@@ -938,9 +938,10 @@ Chat.toDurationString = function (number, options) {
 	const parts = [date.getUTCFullYear() - 1970, date.getUTCMonth(), date.getUTCDate() - 1, date.getUTCHours(), date.getUTCMinutes(), date.getUTCSeconds()];
 	const unitNames = ["second", "minute", "hour", "day", "month", "year"];
 	const positiveIndex = parts.findIndex(elem => elem > 0);
+	const limit = (options && options.limit ? options.limit : 0);
```

Comment:
This is an odd API...

There is a natural choice in the ECMA issue linked in the comment above:

```
minUnit: 'second', // millisecond | second | minute | hour
maxUnit: 'hour' // millisecond | second | minute | hour
```